SDWebImage/SDWebImageDownloader.m

```diff
@@ -23,6 +23,10 @@ @interface SDWebImageDownloadToken ()
 @property (nonatomic, weak, nullable) SDWebImageDownloader *downloader;
 @property (nonatomic, assign, getter=isCancelled) BOOL cancelled;
 
+- (nonnull instancetype)init NS_UNAVAILABLE;
++ (nonnull instancetype)new  NS_UNAVAILABLE;
+- (nonnull instancetype)initWithDownloadOperation:(nullable NSOperation<SDWebImageDownloaderOperation> *)downloadOperation NS_DESIGNATED_INITIALIZER;
```

Comment:
😅Since this is the implementation files, don't need to write `NS_DESIGNATED_INITIALIZER`, because designed initilizer must override super class init method (`NSObject.init`). Just this line is OK.

- (nonnull instancetype)initWithDownloadOperation:(nullable NSOperation<SDWebImageDownloaderOperation> *)downloadOperation;

Reply:
@dreampiggy Interesting, I remember if I mark `init` method `NS_UNAVAILABLE` , no warning would occur previously, maybe apple framework update the interface. 😂 

```
- (instancetype)init
#if NS_ENFORCE_NSOBJECT_DESIGNATED_INITIALIZER
    NS_DESIGNATED_INITIALIZER
#endif
```